gsa/src/web/store/usersettings/defaults/actions.js

```diff
@@ -58,4 +62,36 @@ export const loadUserSettingDefaults = gmp => () => (dispatch, getState) => {
     );
 };
 
+export const loadUserSettingDefault = gmp => id => (dispatch, getState) => {
+  const rootState = getState();
+  const selector = getUserSettingsDefaults(rootState);
+
+  if (selector.isLoading()) {
+    // we are already loading data
+    return Promise.resolve();
+  }
+
+  dispatch(loadingActions.request());
```

Comment:
Personally I wouldn't reuse the SETTINGS**S** actions for loading a single setting. But because updating the actions and reducers for isLoading and error creates a lot of headaches i would accept the current implementation.

But reducing a single settings as settings should override all other settings if i read the code correctly. Therefore this needs to be checked and should get tests!